Test e2e (shard-conformance, HA, snatGW, shared, 1br, ipv6) being cancelled after 2 hours

Test e2e (shard-conformance, HA, snatGW, shared, 1br, ipv6) being cancelled after 2 hours

This test either needs to be adjusted or removed from the suite. It times out.

Somewhere in the middle, we can see plenty of IPv4 tests:
~~~
error:
exit status 1
Retrying...
Aug 24 21:51:39.609: INFO: Running '/usr/local/bin/kubectl --server=https://10.1.1.52:11337 --kubeconfig=/home/runner/admin.conf --namespace=services-4689 exec execpodn2j48 -- /bin/sh -x -c nc -zv -t -w 2 nodeport-update-service 80'
Aug 24 21:51:42.153: INFO: rc: 1
Aug 24 21:51:42.153: INFO: Service reachability failing with error: error running /usr/local/bin/kubectl --server=https://10.1.1.52:11337 --kubeconfig=/home/runner/admin.conf --namespace=services-4689 exec execpodn2j48 -- /bin/sh -x -c nc -zv -t -w 2 nodeport-update-service 80:
Command stdout:
~~~

The test which takes over 5000 seconds:
~~~
2021-08-25T03:49:44.7105161Z {"msg":"FAILED [sig-network] Networking Granular Checks: Services should function for endpoint-Service: udp","total":-1,"completed":3,"skipped":19,"failed":10,"failures":["[sig-network] NetworkPolicy [LinuxOn
ly] NetworkPolicy between server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]","[sig-network] NetworkPolicy [LinuxOnly] Netw
orkPolicy between server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]","[sig-network] NetworkPolicy [LinuxOnly] NetworkPolic
y between server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]","[sig-network] NetworkPolicy [LinuxOnly] NetworkPolicy betwee
n server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]","[sig-network] NetworkPolicy [LinuxOnly] NetworkPolicy between server and client should enforce policy to allow traffic only from a pod in a different namespace based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]","[sig-network] Networking IPerf [Experimental] [Slow] [Feature:Networking-Performance] should transfer ~ 1GB onto the service endpoint 1 servers (maximum of 1 clients)","[sig-network] Networking IPerf [Experimental] [Slow] [Feature:Networking-Performance] should transfer ~ 1GB onto the service endpoint 1 servers (maximum of 1 clients)","[sig-network] Networking IPerf [Experimental] [Slow] [Feature:Networking-Performance] should transfer ~ 1GB onto the service endpoint 1 servers (maximum of 1 clients)","[sig-network] Networking IPerf [Experimental] [Slow] [Feature:Networking-Performance] should transfer ~ 1GB onto the service endpoint 1 servers (maximum of 1 clients)","[sig-network] Networking Granular Checks: Services should function for endpoint-Service: udp"]}
2021-08-25T03:49:44.7152627Z [BeforeEach] [sig-network] Networking
2021-08-25T03:49:44.7188028Z   /go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:185
2021-08-25T03:49:44.7189649Z 
[1mSTEP
[0m: Creating a kubernetes client
2021-08-25T03:49:44.7296311Z Aug 25 02:24:02.641: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:44.7297523Z 
[1mSTEP
[0m: Building a namespace api object, basename nettest
2021-08-25T03:49:44.7298508Z 
[1mSTEP
[0m: Waiting for a default service account to be provisioned in namespace
2021-08-25T03:49:44.7417168Z [BeforeEach] [sig-network] Networking
2021-08-25T03:49:44.7421359Z   /go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/networking.go:81
2021-08-25T03:49:44.7443379Z 
[1mSTEP
[0m: Executing a successful http request from the external internet
2021-08-25T03:49:44.7444685Z [It] should function for endpoint-Service: udp
2021-08-25T03:49:44.7445751Z   /go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/networking.go:254
2021-08-25T03:49:44.7447026Z 
[1mSTEP
[0m: Performing setup for networking test in namespace nettest-1945
2021-08-25T03:49:44.7448009Z 
[1mSTEP
[0m: creating a selector
2021-08-25T03:49:44.7448886Z 
[1mSTEP
[0m: Creating the service pods in kubernetes
2021-08-25T03:49:44.7451179Z Aug 25 02:24:03.072: INFO: Waiting up to 10m0s for all (but 0) nodes to be schedulable
2021-08-25T03:49:44.7454773Z Aug 25 02:24:03.124: INFO: The status of Pod netserver-0 is Pending, waiting for it to be Running (with Ready = true)
2021-08-25T03:49:44.7465021Z Aug 25 02:24:05.128: INFO: The status of Pod netserver-0 is Pending, waiting for it to be Running (with Ready = true)
2021-08-25T03:49:44.7466355Z Aug 25 02:24:07.127: INFO: The status of Pod netserver-0 is Running (Ready = false)
2021-08-25T03:49:44.7467444Z Aug 25 02:24:09.128: INFO: The status of Pod netserver-0 is Running (Ready = false)
2021-08-25T03:49:44.7468539Z Aug 25 02:24:11.127: INFO: The status of Pod netserver-0 is Running (Ready = false)
2021-08-25T03:49:44.7469629Z Aug 25 02:24:13.131: INFO: The status of Pod netserver-0 is Running (Ready = false)
2021-08-25T03:49:44.7472600Z Aug 25 02:24:15.127: INFO: The status of Pod netserver-0 is Running (Ready = false)
2021-08-25T03:49:44.7476777Z Aug 25 02:24:17.127: INFO: The status of Pod netserver-0 is Running (Ready = true)
2021-08-25T03:49:44.7477938Z Aug 25 02:24:17.130: INFO: The status of Pod netserver-1 is Running (Ready = false)
2021-08-25T03:49:44.7480807Z Aug 25 02:24:19.133: INFO: The status of Pod netserver-1 is Running (Ready = false)
(...)
2021-08-25T03:49:44.7492146Z Aug 25 02:24:25.144: INFO: The status of Pod netserver-1 is Running (Ready = true)
2021-08-25T03:49:44.7493023Z Aug 25 02:24:25.156: INFO: The status of Pod netserver-2 is Running (Ready = true)
2021-08-25T03:49:44.7495594Z 
[1mSTEP
[0m: Creating test pods
2021-08-25T03:49:44.7496314Z Aug 25 02:24:29.522: INFO: Setting MaxTries for pod polling to 39 for networking test based on endpoint count 3
2021-08-25T03:49:44.7497141Z 
[1mSTEP
[0m: Getting node addresses
2021-08-25T03:49:44.7497745Z Aug 25 02:24:29.522: INFO: Waiting up to 10m0s for all (but 0) nodes to be schedulable
2021-08-25T03:49:44.7498597Z 
[1mSTEP
[0m: Creating the service on top of the pods in kubernetes
2021-08-25T03:49:44.7499542Z Aug 25 02:24:30.473: INFO: Service node-port-service in namespace nettest-1945 found.
2021-08-25T03:49:44.7500664Z Aug 25 02:24:31.174: INFO: Service session-affinity-service in namespace nettest-1945 found.
2021-08-25T03:49:44.7501662Z 
[1mSTEP
[0m: Waiting for NodePort service to expose endpoint
2021-08-25T03:49:44.7502606Z Aug 25 02:24:32.235: INFO: Waiting for amount of service:node-port-service endpoints to be 3
2021-08-25T03:49:44.7503537Z 
[1mSTEP
[0m: Waiting for Session Affinity service to expose endpoint
2021-08-25T03:49:44.7504587Z Aug 25 02:24:33.418: INFO: Waiting for amount of service:session-affinity-service endpoints to be 3
2021-08-25T03:49:44.7505961Z 
[1mSTEP
[0m: dialing(udp) netserver-0 (endpoint) --> fd00:10:96::4e0a:90 (config.clusterIP)
2021-08-25T03:49:44.7508317Z Aug 25 02:24:33.698: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:44.7509965Z Aug 25 02:24:33.698: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:44.7511699Z Aug 25 02:26:43.648: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
2021-08-25T03:49:44.7514449Z Aug 25 02:26:43.657: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:44.7516064Z Aug 25 02:26:43.657: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:44.7517772Z Aug 25 02:28:54.725: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
(...)
~~~

And this is then stuck here:
~~~
(...)
021-08-25T03:49:44.7532881Z Aug 25 02:33:16.878: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:44.7534497Z Aug 25 02:33:16.878: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:44.7544405Z Aug 25 02:35:27.945: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
2021-08-25T03:49:44.7547246Z Aug 25 02:35:27.960: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:45.5604009Z Aug 25 02:35:27.960: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:45.5607009Z Aug 25 02:37:39.013: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
2021-08-25T03:49:45.5610130Z Aug 25 02:37:39.022: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:45.5611891Z Aug 25 02:37:39.022: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:45.5613739Z Aug 25 02:39:50.081: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
(...)
2021-08-25T03:49:45.5792388Z Aug 25 03:34:26.890: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:45.5793994Z Aug 25 03:34:26.890: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:45.5795691Z Aug 25 03:36:37.952: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
2021-08-25T03:49:45.5798438Z Aug 25 03:36:37.959: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:45.5800051Z Aug 25 03:36:37.959: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:45.5802270Z Aug 25 03:38:49.024: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
2021-08-25T03:49:45.5805083Z Aug 25 03:38:49.032: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:45.5806709Z Aug 25 03:38:49.032: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:45.5808424Z Aug 25 03:41:00.096: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
(...)
2021-08-25T03:49:45.5829821Z Aug 25 03:47:33.321: INFO: ExecWithOptions {Command:[/bin/sh -c curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'] Namespace:nettest-1945 PodName:test-container-pod ContainerName:webserver Stdin:<nil> CaptureStdout:true CaptureStderr:true PreserveWhitespace:false Quiet:false}
2021-08-25T03:49:45.5831442Z Aug 25 03:47:33.321: INFO: >>> kubeConfig: /home/runner/admin.conf
2021-08-25T03:49:45.5833132Z Aug 25 03:49:44.384: INFO: GetResponseFromContainer: failed to execute "curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'": command terminated with exit code 7, stdout: "", stderr: ""
2021-08-25T03:49:45.5834174Z Aug 25 03:49:44.384: INFO: 
2021-08-25T03:49:45.5834876Z Output of kubectl describe pod nettest-1945/netserver-0:
2021-08-25T03:49:45.5835306Z 
2021-08-25T03:49:45.5836580Z Aug 25 03:49:44.384: INFO: Running '/usr/local/bin/kubectl --server=https://10.1.0.164:11337 --kubeconfig=/home/runner/admin.conf --namespace=nettest-1945 describe pod netserver-0 --namespace=nettest-1945'
2021-08-25T03:49:45.5837652Z Aug 25 03:49:44.526: INFO: stderr: ""
2021-08-25T03:49:45.5847232Z Aug 25 03:49:44.526: INFO: stdout: "Name:         netserver-0\nNamespace:    nettest-1945\nPriority:     0\nNode:         ovn-control-plane/fc00:f853:ccd:e793::2\nStart Time:   Wed, 25 Aug 2021 02:24:03 +0000\nLabels:       selector-3657c9a0-8d78-40b8-8fb2-c689a573e481=true\nAnnotations:  k8s.ovn.org/pod-networks:\n                {\"default\":{\"ip_addresses\":[\"fd00:10:244:1::d6/64\"],\"mac_address\":\"0a:58:2a:7c:13:2b\",\"gateway_ips\":[\"fd00:10:244:1::1\"],\"ip_address\":\"fd0...\nStatus:       Running\nIP:           fd00:10:244:1::d6\nIPs:\n  IP:  fd00:10:244:1::d6\nContainers:\n  webserver:\n    Container ID:  containerd://eaad7768a0adb064544ab769862bc0f9cdb6d3ec9886a217525bcb149ab06736\n    Image:         k8s.gcr.io/e2e-test-images/agnhost:2.21\n    Image ID:      k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a\n    Ports:         8080/TCP, 8081/UDP\n    Host Ports:    0/TCP, 0/UDP\n    Args:\n      netexec\n      --http-port=8080\n      --udp-port=8081\n    State:          Running\n      Started:      Wed, 25 Aug 2021 02:24:05 +0000\n    Ready:          True\n    Restart Count:  0\n    Liveness:       http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3\n    Readiness:      http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3\n    Environment:    <none>\n    Mounts:\n      /var/run/secrets/kubernetes.io/serviceaccount from default-token-jrj76 (ro)\nConditions:\n  Type              Status\n  Initialized       True \n  Ready             True \n  ContainersReady   True \n  PodScheduled      True \nVolumes:\n  default-token-jrj76:\n    Type:        Secret (a volume populated by a Secret)\n    SecretName:  default-token-jrj76\n    Optional:    false\nQoS Class:       BestEffort\nNode-Selectors:  kubernetes.io/hostname=ovn-control-plane\nTolerations:     node.kubernetes.io/not-ready:NoExecute op=Exists for 300s\n                 node.kubernetes.io/unreachable:NoExecute op=Exists for 300s\nEvents:          <none>\n"
2021-08-25T03:49:45.5855698Z Aug 25 03:49:44.527: INFO: Name:         netserver-0
2021-08-25T03:49:45.5856345Z Namespace:    nettest-1945
2021-08-25T03:49:45.5856963Z Priority:     0
2021-08-25T03:49:45.5857936Z Node:         ovn-control-plane/fc00:f853:ccd:e793::2
2021-08-25T03:49:45.5858485Z Start Time:   Wed, 25 Aug 2021 02:24:03 +0000
2021-08-25T03:49:45.5859380Z Labels:       selector-3657c9a0-8d78-40b8-8fb2-c689a573e481=true
2021-08-25T03:49:45.5860309Z Annotations:  k8s.ovn.org/pod-networks:
2021-08-25T03:49:45.5861086Z                 {"default":{"ip_addresses":["fd00:10:244:1::d6/64"],"mac_address":"0a:58:2a:7c:13:2b","gateway_ips":["fd00:10:244:1::1"],"ip_address":"fd0...
2021-08-25T03:49:45.5861712Z Status:       Running
2021-08-25T03:49:45.5862084Z IP:           fd00:10:244:1::d6
2021-08-25T03:49:45.5862427Z IPs:
2021-08-25T03:49:45.5862755Z   IP:  fd00:10:244:1::d6
2021-08-25T03:49:45.5863123Z Containers:
2021-08-25T03:49:45.5863490Z   webserver:
2021-08-25T03:49:45.5865022Z     Container ID:  containerd://eaad7768a0adb064544ab769862bc0f9cdb6d3ec9886a217525bcb149ab06736
2021-08-25T03:49:45.5866842Z     Image:         k8s.gcr.io/e2e-test-images/agnhost:2.21
2021-08-25T03:49:45.5868160Z     Image ID:      k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a
2021-08-25T03:49:45.5869157Z     Ports:         8080/TCP, 8081/UDP
2021-08-25T03:49:45.5869565Z     Host Ports:    0/TCP, 0/UDP
2021-08-25T03:49:45.5869938Z     Args:
2021-08-25T03:49:45.5870265Z       netexec
2021-08-25T03:49:45.5870760Z       --http-port=8080
2021-08-25T03:49:45.5871251Z       --udp-port=8081
2021-08-25T03:49:45.5871650Z     State:          Running
2021-08-25T03:49:45.5872063Z       Started:      Wed, 25 Aug 2021 02:24:05 +0000
2021-08-25T03:49:45.5872476Z     Ready:          True
2021-08-25T03:49:45.5872857Z     Restart Count:  0
2021-08-25T03:49:45.5873663Z     Liveness:       http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3
2021-08-25T03:49:45.5874720Z     Readiness:      http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3
2021-08-25T03:49:45.5875363Z     Environment:    <none>
2021-08-25T03:49:45.5875759Z     Mounts:
2021-08-25T03:49:45.5876603Z       /var/run/secrets/kubernetes.io/serviceaccount from default-token-jrj76 (ro)
2021-08-25T03:49:45.5877284Z Conditions:
2021-08-25T03:49:45.5877649Z   Type              Status
2021-08-25T03:49:45.5878058Z   Initialized       True 
2021-08-25T03:49:45.5878468Z   Ready             True 
2021-08-25T03:49:45.5878889Z   ContainersReady   True 
2021-08-25T03:49:45.5879361Z   PodScheduled      True 
2021-08-25T03:49:45.5879755Z Volumes:
2021-08-25T03:49:45.5880306Z   default-token-jrj76:
2021-08-25T03:49:45.5880871Z     Type:        Secret (a volume populated by a Secret)
2021-08-25T03:49:45.5881747Z     SecretName:  default-token-jrj76
2021-08-25T03:49:45.5882255Z     Optional:    false
2021-08-25T03:49:45.5882965Z QoS Class:       BestEffort
2021-08-25T03:49:45.5883786Z Node-Selectors:  kubernetes.io/hostname=ovn-control-plane
2021-08-25T03:49:45.5884915Z Tolerations:     node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
2021-08-25T03:49:45.5885783Z                  node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
2021-08-25T03:49:45.5886362Z Events:          <none>
2021-08-25T03:49:45.5886617Z 
2021-08-25T03:49:45.5886930Z Aug 25 03:49:44.527: INFO: 
2021-08-25T03:49:45.5887632Z Output of kubectl describe pod nettest-1945/netserver-1:
2021-08-25T03:49:45.5888057Z 
2021-08-25T03:49:45.5889332Z Aug 25 03:49:44.527: INFO: Running '/usr/local/bin/kubectl --server=https://10.1.0.164:11337 --kubeconfig=/home/runner/admin.conf --namespace=nettest-1945 describe pod netserver-1 --namespace=nettest-1945'
2021-08-25T03:49:45.5890401Z Aug 25 03:49:44.626: INFO: stderr: ""
2021-08-25T03:49:45.5901597Z Aug 25 03:49:44.626: INFO: stdout: "Name:         netserver-1\nNamespace:    nettest-1945\nPriority:     0\nNode:         ovn-worker/fc00:f853:ccd:e793::4\nStart Time:   Wed, 25 Aug 2021 02:24:03 +0000\nLabels:       selector-3657c9a0-8d78-40b8-8fb2-c689a573e481=true\nAnnotations:  k8s.ovn.org/pod-networks:\n                {\"default\":{\"ip_addresses\":[\"fd00:10:244:3::ce/64\"],\"mac_address\":\"0a:58:cd:67:70:b3\",\"gateway_ips\":[\"fd00:10:244:3::1\"],\"ip_address\":\"fd0...\nStatus:       Running\nIP:           fd00:10:244:3::ce\nIPs:\n  IP:  fd00:10:244:3::ce\nContainers:\n  webserver:\n    Container ID:  containerd://4b8aebe3698588371dc200bc02da235348084e54c8e57456a748a32077daf920\n    Image:         k8s.gcr.io/e2e-test-images/agnhost:2.21\n    Image ID:      k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a\n    Ports:         8080/TCP, 8081/UDP\n    Host Ports:    0/TCP, 0/UDP\n    Args:\n      netexec\n      --http-port=8080\n      --udp-port=8081\n    State:          Running\n      Started:      Wed, 25 Aug 2021 02:24:05 +0000\n    Ready:          True\n    Restart Count:  0\n    Liveness:       http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3\n    Readiness:      http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3\n    Environment:    <none>\n    Mounts:\n      /var/run/secrets/kubernetes.io/serviceaccount from default-token-jrj76 (ro)\nConditions:\n  Type              Status\n  Initialized       True \n  Ready             True \n  ContainersReady   True \n  PodScheduled      True \nVolumes:\n  default-token-jrj76:\n    Type:        Secret (a volume populated by a Secret)\n    SecretName:  default-token-jrj76\n    Optional:    false\nQoS Class:       BestEffort\nNode-Selectors:  kubernetes.io/hostname=ovn-worker\nTolerations:     node.kubernetes.io/not-ready:NoExecute op=Exists for 300s\n                 node.kubernetes.io/unreachable:NoExecute op=Exists for 300s\nEvents:          <none>\n"
2021-08-25T03:49:45.5909124Z Aug 25 03:49:44.626: INFO: Name:         netserver-1
2021-08-25T03:49:45.5909740Z Namespace:    nettest-1945
2021-08-25T03:49:45.5910173Z Priority:     0
2021-08-25T03:49:45.5910898Z Node:         ovn-worker/fc00:f853:ccd:e793::4
2021-08-25T03:49:45.5911379Z Start Time:   Wed, 25 Aug 2021 02:24:03 +0000
2021-08-25T03:49:45.5912259Z Labels:       selector-3657c9a0-8d78-40b8-8fb2-c689a573e481=true
2021-08-25T03:49:45.5913209Z Annotations:  k8s.ovn.org/pod-networks:
2021-08-25T03:49:45.5913995Z                 {"default":{"ip_addresses":["fd00:10:244:3::ce/64"],"mac_address":"0a:58:cd:67:70:b3","gateway_ips":["fd00:10:244:3::1"],"ip_address":"fd0...
2021-08-25T03:49:45.5914607Z Status:       Running
2021-08-25T03:49:45.5914996Z IP:           fd00:10:244:3::ce
2021-08-25T03:49:45.5915324Z IPs:
2021-08-25T03:49:45.5915660Z   IP:  fd00:10:244:3::ce
2021-08-25T03:49:45.5916016Z Containers:
2021-08-25T03:49:45.5916501Z   webserver:
2021-08-25T03:49:45.5917907Z     Container ID:  containerd://4b8aebe3698588371dc200bc02da235348084e54c8e57456a748a32077daf920
2021-08-25T03:49:45.5919622Z     Image:         k8s.gcr.io/e2e-test-images/agnhost:2.21
2021-08-25T03:49:45.5920955Z     Image ID:      k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a
2021-08-25T03:49:45.5922011Z     Ports:         8080/TCP, 8081/UDP
2021-08-25T03:49:45.5922436Z     Host Ports:    0/TCP, 0/UDP
2021-08-25T03:49:45.5922795Z     Args:
2021-08-25T03:49:45.5923135Z       netexec
2021-08-25T03:49:45.5923622Z       --http-port=8080
2021-08-25T03:49:45.5924131Z       --udp-port=8081
2021-08-25T03:49:45.5924506Z     State:          Running
2021-08-25T03:49:45.5924941Z       Started:      Wed, 25 Aug 2021 02:24:05 +0000
2021-08-25T03:49:45.5925354Z     Ready:          True
2021-08-25T03:49:45.5925722Z     Restart Count:  0
2021-08-25T03:49:45.5926538Z     Liveness:       http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3
2021-08-25T03:49:45.5927575Z     Readiness:      http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3
2021-08-25T03:49:45.5928229Z     Environment:    <none>
2021-08-25T03:49:45.5928605Z     Mounts:
2021-08-25T03:49:45.5929463Z       /var/run/secrets/kubernetes.io/serviceaccount from default-token-jrj76 (ro)
2021-08-25T03:49:45.5930124Z Conditions:
2021-08-25T03:49:45.5930502Z   Type              Status
2021-08-25T03:49:45.5930909Z   Initialized       True 
2021-08-25T03:49:45.5931293Z   Ready             True 
2021-08-25T03:49:45.5931727Z   ContainersReady   True 
2021-08-25T03:49:45.5932185Z   PodScheduled      True 
2021-08-25T03:49:45.5932581Z Volumes:
2021-08-25T03:49:45.5933116Z   default-token-jrj76:
2021-08-25T03:49:45.5933683Z     Type:        Secret (a volume populated by a Secret)
2021-08-25T03:49:45.5934407Z     SecretName:  default-token-jrj76
2021-08-25T03:49:45.5934924Z     Optional:    false
2021-08-25T03:49:45.5935525Z QoS Class:       BestEffort
2021-08-25T03:49:45.5936265Z Node-Selectors:  kubernetes.io/hostname=ovn-worker
2021-08-25T03:49:45.5937212Z Tolerations:     node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
2021-08-25T03:49:45.5938082Z                  node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
2021-08-25T03:49:45.5938683Z Events:          <none>
2021-08-25T03:49:45.5938920Z 
2021-08-25T03:49:45.5939942Z Output of kubectl describe pod nettest-1945/netserver-2:
2021-08-25T03:49:45.5940354Z 
2021-08-25T03:49:45.5941825Z Aug 25 03:49:44.626: INFO: Running '/usr/local/bin/kubectl --server=https://10.1.0.164:11337 --kubeconfig=/home/runner/admin.conf --namespace=nettest-1945 describe pod netserver-2 --namespace=nettest-1945'
2021-08-25T03:49:45.5942897Z Aug 25 03:49:44.695: INFO: stderr: ""
2021-08-25T03:49:45.5952275Z Aug 25 03:49:44.695: INFO: stdout: "Name:         netserver-2\nNamespace:    nettest-1945\nPriority:     0\nNode:         ovn-worker2/fc00:f853:ccd:e793::3\nStart Time:   Wed, 25 Aug 2021 02:24:03 +0000\nLabels:       selector-3657c9a0-8d78-40b8-8fb2-c689a573e481=true\nAnnotations:  k8s.ovn.org/pod-networks:\n                {\"default\":{\"ip_addresses\":[\"fd00:10:244:2::df/64\"],\"mac_address\":\"0a:58:28:b5:18:69\",\"gateway_ips\":[\"fd00:10:244:2::1\"],\"ip_address\":\"fd0...\nStatus:       Running\nIP:           fd00:10:244:2::df\nIPs:\n  IP:  fd00:10:244:2::df\nContainers:\n  webserver:\n    Container ID:  containerd://9ac69599c35e9f798c22801ebfa16bc9208703c351c40691b3a58b8b0672a3cd\n    Image:         k8s.gcr.io/e2e-test-images/agnhost:2.21\n    Image ID:      k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a\n    Ports:         8080/TCP, 8081/UDP\n    Host Ports:    0/TCP, 0/UDP\n    Args:\n      netexec\n      --http-port=8080\n      --udp-port=8081\n    State:          Running\n      Started:      Wed, 25 Aug 2021 02:24:05 +0000\n    Ready:          True\n    Restart Count:  0\n    Liveness:       http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3\n    Readiness:      http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3\n    Environment:    <none>\n    Mounts:\n      /var/run/secrets/kubernetes.io/serviceaccount from default-token-jrj76 (ro)\nConditions:\n  Type              Status\n  Initialized       True \n  Ready             True \n  ContainersReady   True \n  PodScheduled      True \nVolumes:\n  default-token-jrj76:\n    Type:        Secret (a volume populated by a Secret)\n    SecretName:  default-token-jrj76\n    Optional:    false\nQoS Class:       BestEffort\nNode-Selectors:  kubernetes.io/hostname=ovn-worker2\nTolerations:     node.kubernetes.io/not-ready:NoExecute op=Exists for 300s\n                 node.kubernetes.io/unreachable:NoExecute op=Exists for 300s\nEvents:          <none>\n"
2021-08-25T03:49:45.5960449Z Aug 25 03:49:44.695: INFO: Name:         netserver-2
2021-08-25T03:49:45.5961080Z Namespace:    nettest-1945
2021-08-25T03:49:45.5961513Z Priority:     0
2021-08-25T03:49:45.5962241Z Node:         ovn-worker2/fc00:f853:ccd:e793::3
2021-08-25T03:49:45.5962725Z Start Time:   Wed, 25 Aug 2021 02:24:03 +0000
2021-08-25T03:49:45.5963610Z Labels:       selector-3657c9a0-8d78-40b8-8fb2-c689a573e481=true
2021-08-25T03:49:45.5964658Z Annotations:  k8s.ovn.org/pod-networks:
2021-08-25T03:49:45.5965429Z                 {"default":{"ip_addresses":["fd00:10:244:2::df/64"],"mac_address":"0a:58:28:b5:18:69","gateway_ips":["fd00:10:244:2::1"],"ip_address":"fd0...
2021-08-25T03:49:45.5966032Z Status:       Running
2021-08-25T03:49:45.5966415Z IP:           fd00:10:244:2::df
2021-08-25T03:49:45.5966753Z IPs:
2021-08-25T03:49:45.5967075Z   IP:  fd00:10:244:2::df
2021-08-25T03:49:45.5967439Z Containers:
2021-08-25T03:49:45.5967806Z   webserver:
2021-08-25T03:49:45.5969218Z     Container ID:  containerd://9ac69599c35e9f798c22801ebfa16bc9208703c351c40691b3a58b8b0672a3cd
2021-08-25T03:49:45.5970906Z     Image:         k8s.gcr.io/e2e-test-images/agnhost:2.21
2021-08-25T03:49:45.5972242Z     Image ID:      k8s.gcr.io/e2e-test-images/agnhost@sha256:ab055cd3d45f50b90732c14593a5bf50f210871bb4f91994c756fc22db6d922a
2021-08-25T03:49:45.5973222Z     Ports:         8080/TCP, 8081/UDP
2021-08-25T03:49:45.5973649Z     Host Ports:    0/TCP, 0/UDP
2021-08-25T03:49:45.5974021Z     Args:
2021-08-25T03:49:45.5974349Z       netexec
2021-08-25T03:49:45.5974847Z       --http-port=8080
2021-08-25T03:49:45.5975508Z       --udp-port=8081
2021-08-25T03:49:45.5975906Z     State:          Running
2021-08-25T03:49:45.5976319Z       Started:      Wed, 25 Aug 2021 02:24:05 +0000
2021-08-25T03:49:45.5976724Z     Ready:          True
2021-08-25T03:49:45.5977088Z     Restart Count:  0
2021-08-25T03:49:45.5977910Z     Liveness:       http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3
2021-08-25T03:49:45.5978940Z     Readiness:      http-get http://:8080/healthz delay=10s timeout=30s period=10s #success=1 #failure=3
2021-08-25T03:49:45.5979606Z     Environment:    <none>
2021-08-25T03:49:45.5979993Z     Mounts:
2021-08-25T03:49:45.5980836Z       /var/run/secrets/kubernetes.io/serviceaccount from default-token-jrj76 (ro)
2021-08-25T03:49:45.5981507Z Conditions:
2021-08-25T03:49:45.5981871Z   Type              Status
2021-08-25T03:49:45.5982285Z   Initialized       True 
2021-08-25T03:49:45.5982670Z   Ready             True 
2021-08-25T03:49:45.5983102Z   ContainersReady   True 
2021-08-25T03:49:45.5983560Z   PodScheduled      True 
2021-08-25T03:49:45.5983956Z Volumes:
2021-08-25T03:49:45.5984628Z   default-token-jrj76:
2021-08-25T03:49:45.5985206Z     Type:        Secret (a volume populated by a Secret)
2021-08-25T03:49:45.5985951Z     SecretName:  default-token-jrj76
2021-08-25T03:49:45.5986458Z     Optional:    false
2021-08-25T03:49:45.5986880Z QoS Class:       BestEffort
2021-08-25T03:49:45.5987606Z Node-Selectors:  kubernetes.io/hostname=ovn-worker2
2021-08-25T03:49:45.5988586Z Tolerations:     node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
2021-08-25T03:49:45.5989563Z                  node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
2021-08-25T03:49:45.5990279Z Events:          <none>
2021-08-25T03:49:45.5990535Z 
2021-08-25T03:49:45.5991095Z Aug 25 03:49:44.695: INFO: encountered error during dial (did not find expected responses... 
2021-08-25T03:49:45.5991755Z Tries 39
2021-08-25T03:49:45.5992756Z Command curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'
2021-08-25T03:49:45.5993409Z retrieved map[]
2021-08-25T03:49:45.5994120Z expected map[netserver-0:{} netserver-1:{} netserver-2:{}])
2021-08-25T03:49:45.5994906Z Aug 25 03:49:44.695: FAIL: failed dialing endpoint, did not find expected responses... 
2021-08-25T03:49:45.5995471Z Tries 39
2021-08-25T03:49:45.5996433Z Command curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'
2021-08-25T03:49:45.5997330Z retrieved map[]
2021-08-25T03:49:45.5998217Z expected map[netserver-0:{} netserver-1:{} netserver-2:{}]
2021-08-25T03:49:45.5998662Z 
2021-08-25T03:49:45.5999001Z Full Stack Trace
2021-08-25T03:49:45.5999510Z k8s.io/kubernetes/test/e2e.RunE2ETests(0xc002260180)
2021-08-25T03:49:45.6000181Z    _output/local/go/src/k8s.io/kubernetes/test/e2e/e2e.go:130 +0x36c
2021-08-25T03:49:45.6000804Z k8s.io/kubernetes/test/e2e.TestE2E(0xc002260180)
2021-08-25T03:49:45.6001457Z    _output/local/go/src/k8s.io/kubernetes/test/e2e/e2e_test.go:144 +0x2b
2021-08-25T03:49:45.6002070Z testing.tRunner(0xc002260180, 0x4eef290)
2021-08-25T03:49:45.6002675Z    /usr/local/go/src/testing/testing.go:1194 +0xef
2021-08-25T03:49:45.6003177Z created by testing.(*T).Run
2021-08-25T03:49:45.6003700Z    /usr/local/go/src/testing/testing.go:1239 +0x2b3
2021-08-25T03:49:45.6004401Z [AfterEach] [sig-network] Networking
2021-08-25T03:49:45.6005199Z   /go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:186
2021-08-25T03:49:45.6006000Z Aug 25 03:49:44.696: INFO: Waiting up to 3m0s for all (but 0) nodes to be ready
2021-08-25T03:49:45.6006828Z 
[1mSTEP
[0m: Destroying namespace "nettest-1945" for this suite.
2021-08-25T03:49:45.6007252Z 
2021-08-25T03:49:45.6007439Z 
2021-08-25T03:49:45.6007962Z 
[91m
 Failure [5142.066 seconds]
2021-08-25T03:49:45.6008526Z [sig-network] Networking
2021-08-25T03:49:45.6009426Z 
[90m/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/framework.go:23
2021-08-25T03:49:45.6010136Z   Granular Checks: Services
2021-08-25T03:49:45.6011038Z   
[90m/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/networking.go:149
2021-08-25T03:49:45.6012019Z     
[91m
[1mshould function for endpoint-Service: udp [It]
2021-08-25T03:49:45.6013032Z     
[90m/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/network/networking.go:254
2021-08-25T03:49:45.6013573Z 
2021-08-25T03:49:45.6014300Z     
[91mAug 25 03:49:44.695: failed dialing endpoint, did not find expected responses... 
2021-08-25T03:49:45.6014855Z     Tries 39
2021-08-25T03:49:45.6015952Z     Command curl -g -q -s 'http://[fd00:10:244:1::d6]:8080/dial?request=hostname&protocol=udp&host=fd00:10:96::4e0a&port=90&tries=1'
2021-08-25T03:49:45.6016604Z     retrieved map[]
2021-08-25T03:49:45.6017343Z     expected map[netserver-0:{} netserver-1:{} netserver-2:{}]
2021-08-25T03:49:45.6017785Z 
2021-08-25T03:49:45.6018540Z     /go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/onsi/ginkgo/internal/leafnodes/runner.go:113
2021-08-25T03:49:45.6019469Z 
[90m------------------------------
2021-08-25T03:50:53.8446415Z ##[error]The operation was canceled.
~~~

--- a/go-controller/pkg/node/gateway_shared_intf.go
+++ b/go-controller/pkg/node/gateway_shared_intf.go
@@ -747,7 +747,7 @@ func flowsForDefaultBridge(ofPortPhys, bridgeMacAddress, ofPortPatch, ofPortHost
 		dftFlows = append(dftFlows,
 			fmt.Sprintf("cookie=%s, priority=205, in_port=%s, dl_dst=%s, udp6, udp_dst=%d, "+
 				"actions=output:%s", defaultOpenFlowCookie, ofPortPhys, bridgeMacAddress, config.Default.EncapPort,
-				ofPortPhys))
+				ofPortHost))
 		// perform NORMAL action otherwise.
 		dftFlows = append(dftFlows,
 			fmt.Sprintf("cookie=%s, priority=200, in_port=%s, udp6, udp_dst=%d, "+

